experimental/devices/cap1188/cap1188_test.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed under the Apache License, Version 2.0
 // that can be found in the LICENSE file.
 
-package cap1188_test
+package cap1188
```

Comment:
I suggest we keep the examples in a test package so the code can be copied/pasted directly.

Reply:
It must be in the same package to be able to override "sleep"

Reply:
Maybe we can move the example to its own file then, it will still have a sleep but it's only 1 example. Thoughts?

Reply:
Yes good idea, done.